pkg/webui/console/views/device-add/repository/device-registration/index.js

```diff
@@ -69,81 +69,29 @@ const Registration = props => {
 
   return (
     <Overlay visible={fetching} loading={fetching} spinnerMessage={m.fetching}>
-      <FreqPlansSelect
-        required
-        glossaryId={glossaryId.FREQUENCY_PLAN}
-        name="frequency_plan_id"
-        bandId={band}
-      />
-      {isOTAA && (
-        <>
-          <Form.Field
-            title={lwVersion < 104 ? sharedMessages.appEUI : sharedMessages.joinEUI}
-            component={JoinEUIPRefixesInput}
-            name="ids.join_eui"
-            description={
-              lwVersion < 104 ? sharedMessages.appEUIDescription : sharedMessages.joinEUIDescription
-            }
-            prefixes={prefixes}
-            required
-            showPrefixes
-            glossaryId={lwVersion < 104 ? glossaryId.APP_EUI : glossaryId.JOIN_EUI}
-          />
-          <Form.Field
-            title={sharedMessages.devEUI}
-            name="ids.dev_eui"
-            type="byte"
-            min={8}
-            max={8}
-            description={sharedMessages.deviceEUIDescription}
-            required
-            component={Input}
-            glossaryId={glossaryId.DEV_EUI}
-          />
-          <Form.Field
-            required
-            title={sharedMessages.appKey}
-            name="root_keys.app_key.key"
-            type="byte"
-            min={16}
-            max={16}
-            description={
-              lwVersion >= 110
-                ? sharedMessages.appKeyNewDescription
-                : sharedMessages.appKeyDescription
-            }
-            component={Input.Generate}
-            disabled={!mayEditKeys}
-            mayGenerateValue={mayEditKeys}
-            onGenerateValue={generate16BytesKey}
-            glossaryId={glossaryId.APP_KEY}
-          />
-          {lwVersion >= 110 && (
+      <div data-test-id="device-registration">
```

Comment:
The diff confuses me. Was there anything changed apart from wrapping everything into a div?

Reply:
No, only the wrapping div.